lldb/source/DataFormatters/FormatManager.cpp

```diff
@@ -451,8 +451,13 @@ bool FormatManager::ShouldPrintAsOneLiner(ValueObject &valobj) {
   if (valobj.GetSummaryFormat().get() != nullptr)
     return valobj.GetSummaryFormat()->IsOneLiner();
 
+  auto num_children = valobj.GetNumChildren();
+  if (!num_children) {
+    llvm::consumeError(num_children.takeError());
```

Comment:
could you log this some channel instead of consuming the error? That would still be NFC and would fit well within this PR

Reply:
Done!